entry.go

```diff
@@ -223,6 +254,53 @@ func (te *TunaEntry) StartReverse(stream *smux.Stream) error {
 	return nil
 }
 
+func (te *TunaEntry) nanoPayClaim(stream *smux.Stream, npc *nkn.NanoPayClaimer, reverseBytesIn, reverseBytesOut []uint64, lastComputed, lastClaimed common.Fixed64, lastUpdate time.Time) {
+
+	txData, err := ioutil.ReadAll(stream)
+	if err != nil && err.Error() != io.EOF.Error() {
+		log.Println("Couldn't read payment stream:", err)
+		return
+	}
+	if len(txData) == 0 {
+		return
+	}
+	tx := new(transaction.Transaction)
+	if err := tx.Unmarshal(txData); err != nil {
+		log.Println("Couldn't unmarshal payment stream data:", err)
+		return
+	}
+	//payload, _ := transaction.Unpack(tx.UnsignedTx.Payload)
+	//fmt.Println("claim tx amount:", payload.(*pb.NanoPay).Amount)
+	amount, err := npc.Claim(tx)
+	if err != nil {
+		log.Println("Couldn't accept nano pay update:", err)
+		return
+	}
+
+	totalCost := common.Fixed64(0)
+	for i := range reverseBytesIn {
+		in := atomic.LoadUint64(&reverseBytesIn[i])
+		out := atomic.LoadUint64(&reverseBytesOut[i])
+		if in == 0 && out == 0 {
+			continue
+		}
+		//service, err := te.getService(byte(i))
+		if err != nil {
+			continue
+		}
+		entryToExitPrice, exitToEntryPrice := common.Fixed64(100000), common.Fixed64(100000)
+		if err != nil {
+			continue
+		}
+		totalCost += entryToExitPrice*common.Fixed64(in)/TrafficUnit + exitToEntryPrice*common.Fixed64(out)/TrafficUnit
```

Comment:
For forward, in means entry to exit
For exit, in means exit to entry
So we should use `entryToExitBytes` and `exitToEntryBytes` instead of `bytesIn` and `bytesOut` in this function